### PR TITLE
py-agate-sql: update to 0.5.3

### PR DESCRIPTION
**Pull request by chrstphrchvz**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.14.2 18C54
Xcode 10.1 10B61
Python 2.7.15

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] ~~referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?~~

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-agate-sql/Portfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 
 set base_name       agate-sql
 name                py-agate-sql
-version             0.5.2
+version             0.5.3
 python.versions     27 35 36
```
Comment (pmetzger) [inline python/py-agate-sql/Portfile]: You might want to check on whether 3.7 is now supported.

**Comment by pmetzger** [comment]
Argh! I didn't notice this wasn't `openmaintainer`. @esafak, was it okay anyway?

File: python/py-agate-sql/Portfile (hunk 1 above)
Comment (chrstphrchvz) [reply python/py-agate-sql/Portfile]: The developer mentions support for 3.3+ and 3.6, but I have not found any mention of 3.7 specifically (same goes for csvkit and other dependencies: agate, agate-excel, agate-dbf). Some other package managers (Anaconda, Fedora) already have it available for 3.7 though. Should a subport be added if it works?

**Comment by esafak** [comment]
> Argh! I didn't notice this wasn't `openmaintainer`. @esafak, was it okay anyway?

Make it openmaintaner, and add 3.7 if it works. I have no time to manage it.

**Comment by pmetzger** [comment]
@chrstphrchvz Please update as @esafak requests.

**Pull request merged**